iocage_lib/ioc_stop.py

```diff
@@ -87,6 +87,23 @@ def __stop_jail__(self):
             _callback=self.callback,
             silent=self.silent)
 
+        rctl_jail = iocage_lib.ioc_json.IOCRCTL(self.uuid)
+        if rctl_jail.rctl_rules_exist():
+            failed = rctl_jail.remove_rctl_rules()
+            if failed:
+                msg = 'Failed to remove'
+            else:
+                msg = 'Successfully removed'
+
+            iocage_lib.ioc_common.logit(
+                {
+                    'level': 'INFO',
```

Comment:
This should become an error if they failed to be removed

Reply:
I think I can have this covered by the force flag, so if for any reason they fail, force would do an override anyways